### PR TITLE
Switch to GitHub Actions

### PR DESCRIPTION
**Pull request by Fryguy**
https://github.com/Fryguy/manageiq-ssh-util/actions?query=branch%3Agithub_actions for CI status.

### CODE REVIEW

**Comment by bdunne** [comment]
Is this ready to go?

**Review by kbrock — APPROVED** [review]
CI all things

**Comment by Fryguy** [comment]
> Is this ready to go?

I am review them one-by-one which is why I WIP'd them...I've found I need manual changes in most of them (even if they go green).

File: lib/manageiq/ssh/util.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 require 'net/ssh'
 require 'net/sftp'
 require 'tempfile'
+require 'active_support'
```
Comment (djberg96) [inline lib/manageiq/ssh/util.rb:4]: I'm curious when/why this became a requirement, but https://github.com/rails/rails/issues/43889 suggests that it's necessary.

Comment (Fryguy) [reply lib/manageiq/ssh/util.rb:4]: That's the one

**Review by agrare — APPROVED** [review]

**Pull request merged**